### PR TITLE
[CORL-2090] Comment Seen

### PR DESCRIPTION
**Pull request by alinhle**
## What does this PR do?
- Introduce Comment Seen concept
- Visually marks comments that have not yet been rendered to the use
- Data is saved to indexedDB
- Replaces `PymStorage` with  `PostMessageStorage`
- Introduces `indexedDBStorage` as new backend storage for `PostMessageStorage`
- `useCommentSeen` hook.

## How do I test this PR?
- Activate `COMMENT_SEEN` feature flag

## TODO
- [x] Design
- ~Events / Tracking~

### CODE REVIEW

File: src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx [hunk 1]
```diff
@@ -362,6 +372,11 @@ export const CommentContainer: FunctionComponent<Props> = ({
     >
       <HorizontalGutter>
         <IndentedComment
+          classNameIndented={cn({
+            [styles.commentSeenEnabled]: commentSeenEnabled,
+            [styles.notSeen]: shouldApplyNotSeenClass,
+            "coral-comment-notSeen": shouldApplyNotSeenClass,
```
Comment (tessalt) [inline src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx]: shouldn't this be in the CLASSES.ts file?

Comment (nick-funk) [reply src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx]: Yeah, would agree, this seems odd to be hard coded here

File: src/core/client/framework/lib/bootstrap/createManaged.tsx [hunk 2]
```diff
@@ -302,14 +307,31 @@ function resolveLocalStorage(pym?: PymChild): PromisifiedStorage {
...
  */
-function resolveSessionStorage(pym?: PymChild): PromisifiedStorage {
-  if (pym && areWeInIframe()) {
-    // Use session storage over pym when we have pym and are in an iframe.
-    return createPymStorage(pym, "sessionStorage");
+function resolveSessionStorage(
+  postMessage?: PostMessageService,
+  pym?: PymChild
+): PromisifiedStorage {
+  if (pym && postMessage && areWeInIframe()) {
```
Comment (alinhle) [inline src/core/client/framework/lib/bootstrap/createManaged.tsx]: Note to self: Do not need to check for pym here

**Review by nick-funk — APPROVED** [review]
Tested and it appears to be working very well!

**Comment by alinhle** [comment]
Done

**Review by marcushaddon — APPROVED** [review]

**Comment by yogeshbeniwal** [comment]
> * Activate `COMMENT_SEEN` feature flag

@cvle Can you pls help with mutation to use to enable this feature?

**Pull request merged**